### ISSUE TITLE
Bug: Unable to download attachments with same name

### ISSUE THREAD

**Issue by vipranarayan14** [Foundry376/Mailspring#1901]
**Describe the bug**
From an email, the attachments with the same name are not downloading. Only one of or the last of the attachments is downloading.

**To Reproduce**
1. Create two files with the same filename, say 'a.txt', in two different folders.
2. Compose a new email.
3. Add the two files from the two folders as attachments.
4. Send the mail to yourself.
5. Now, open the received email.
6. Click the "Download All" button above the attachments, and select the target folder.
7. Now, open the target folder. 
Only one of the attachments will be there i.e. only one 'a.txt' file will be there.

**Expected behavior**
When files with the same filenames are downloaded, all the files should be there in the target folder. Their names should be suffixed with increasing numbers, for example, 'a-1.txt', 'a-2.txt', etc.

**Setup: (please complete the following information):**
 - OS: macOS
 - OS Version: 10.15.4
 - Mailspring Version: 1.7.4

**Additional context** 
I browsed and filtered through the previous issues as far as I could but did not find any similar one.

**Comment by bengotow** [comment]
Hey @vipranarayan14 — thanks for reporting this! I think you're right that this is the first time this has come up, but it's definitely a bug. When you add an attachment, Mailspring copies the file to a temporary directory (so that saving it after attaching it or deleting it doesn't block you from sending the message.) I think that right now the file names aren't modified - will see if I can get this fixed since it should be quick.

**Comment by vipranarayan14** [comment]
Thank you for fixing the issue so quickly. May I know when the fix will be released?

BTW, I tried to clone the repo to fix the issue myself and give a pull request. I followed the [installation instructions](https://github.com/Foundry376/Mailspring/blob/master/CONTRIBUTING.md#build-and-run-from-source) given in contributing guidelines. But the `npm install` command took too long. The installation was stuck at 
```bash
-- Running npm install in ./app with electron config --
``` 
while executing the post-install script:

```bash
node scripts/postinstall.js
```
 I couldn't proceed from there.
